clients/src/main/java/org/apache/kafka/clients/ClientUtils.java

```diff
@@ -46,7 +46,7 @@
                 try {
                     InetSocketAddress address = new InetSocketAddress(host, port);
                     if (address.isUnresolved())
-                        throw new ConfigException("DNS resolution failed for url in " + CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG + ": " + url);
+                        log.error("DNS resolution failed for url in " + CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG + ": " + url);
                     addresses.add(address);
```

Comment:
@lindong28 I've been trying to less strict, but I think you're right. I've updated my PR.